src/clr-addons/notification/notification.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { animate, style, transition, trigger, state } from '@angular/animations';
+import { timer, Subscription } from 'rxjs';
+import { interval } from 'rxjs';
+import { takeWhile } from 'rxjs/operators';
+
+const initState = { value: 'currentPositon', params: { percents: 0 } };
+
+@Component({
+  selector: 'clr-notification',
+  templateUrl: './notification.html',
+  animations: [
+    trigger('slideDown', [
+      transition(':enter', [
+        style({ transform: 'translateY(-1000%)' }),
+        animate('0.7s', style({ transform: 'translateY(0%)' })),
+      ]),
+    ]),
+    trigger('down', [
+      state('currentPosition', style({ transform: 'translateY({{percents}}%)' }), { params: { percents: '0' } }),
+      state('moveDown', style({ transform: 'translateY({{percents}}%)' }), { params: { percents: '0' } }),
+      transition('* => moveDown', animate('0.3s ease-in')),
+    ]),
+    trigger('fade', [transition(':leave', [animate('0.5s ease-in-out', style({ opacity: 0 }))])]),
+  ],
+  host: {
+    '[class.notification]': 'true',
+  },
+})
+export class ClrNotification implements OnInit {
+  _open: boolean = false;
```

Comment:
underscore is just needed for public variables, which should not be considered public by the consumer.
so this should be "privated opened" (name clash)